apps/earn-protocol-landing-page/docker/Dockerfile

```
@@ -0,0 +1,48 @@
+FROM node:20-alpine
+EXPOSE 3000
+WORKDIR .
```

Hadolint

[error] Use absolute WORKDIR

(DL3000)